proto/info/v1/service.proto

```diff
@@ -0,0 +1,126 @@
+syntax = "proto3";
+package info.v1;
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/ava-labs/avalanchego/proto/pb/info/v1;infov1";
+
+// InfoService is the API service for unprivileged info on a node
+service InfoService {
+  rpc GetNodeVersion(GetNodeVersionRequest) returns (GetNodeVersionResponse);
+  rpc GetNodeID(GetNodeIDRequest) returns (GetNodeIDResponse);
+  rpc GetNodeIP(GetNodeIPRequest) returns (GetNodeIPResponse);
+  rpc GetNetworkID(GetNetworkIDRequest) returns (GetNetworkIDResponse);
+  rpc GetNetworkName(GetNetworkNameRequest) returns (GetNetworkNameResponse);
+  rpc GetBlockchainID(GetBlockchainIDRequest) returns (GetBlockchainIDResponse);
+  rpc Peers(PeersRequest) returns (PeersResponse);
+  rpc IsBootstrapped(IsBootstrappedRequest) returns (IsBootstrappedResponse);
+  rpc Upgrades(UpgradesRequest) returns (UpgradesResponse);
+  rpc Uptime(UptimeRequest) returns (UptimeResponse);
+  rpc GetVMs(GetVMsRequest) returns (GetVMsResponse);
```

Comment:
Some of these are inconsistent with others in terms of naming. It seems like the convention for rpcs is to just have rpcs called `Foo` instead of `GetFoo` unless we are disambiguating like `GetFooByBar` or `GetFooByBaz` (i.e the Get is implied and is otherwise redundant). Could we drop the `Get*` prefix?